.github/workflows/release.yml

```diff
@@ -60,12 +64,24 @@ jobs:
         id: go
       - name: Check out code into the Go module directory
         uses: actions/checkout@v1
+      - name: Install Android NDK
+        run: |
+          curl -L https://dl.google.com/android/repository/android-ndk-r21e-linux-x86_64.zip -o android-ndk.zip
+          unzip android-ndk.zip -d $HOME/android-ndk
+          export ANDROID_NDK_HOME=$HOME/android-ndk/android-ndk-r21e
+          export PATH=$PATH:$ANDROID_NDK_HOME/toolchains/llvm/prebuilt/linux-x86_64/bin
+        shell: bash
       - name: Build the Go binary
-        run: go build
+        run: |
+          if [ "${{ matrix.goos }}" = "windows" ]; then
+            env GOOS=windows GOARCH=amd64 go build
+          elif [ "${{ matrix.goos }}" = "android" ]; then
+            env GOOS=android GOARCH=arm CGO_ENABLED=1 CC=$ANDROID_NDK_HOME/toolchains/llvm/prebuilt/linux-x86_64/bin/armv7a-linux-androideabi21-clang go build
+          else
+            go build
+          fi
         env:
-          GOOS: ${{ matrix.goos }}
-          GOARCH: ${{ matrix.goarch }}
-          GOARM: ${{ matrix.goarm }}
+          CGO_ENABLED: 1
```

Comment:
can we keep the `env` variables here instead of using the `env` command in the `run` section?

Reply:
run section only usages env variables for `arm` build if we set in `env` section it'll become global `env` and will result in failure of other os builds

Reply:
this is wrong.

Reply:
> this is wrong.

@wargio what is wrong sir? any suggestions 🤔

Reply:
GOOS, GOARCH and GOARM should stay there. they are required to build correctly the bins
if you don't like the matrix we can change it to a list of include.

Reply:
Hmm... Wait let me try by bdoing so if runs are success then no problems

Reply:
@wargio doing so led to [failures](https://github.com/AbhiTheModder/rz-pm/actions/runs/6558779742/job/17813062242) 

Trying with includes suggestion now😓

Reply:
@wargio even includes isn't working... now I'm backing off... maybe will try later :(

Reply:
Instead of adding `arm` to `goarch`, just  add:
```
          - goos: android
            goarch: arm
AbhiTheModder marked this conversation as resolved.
            asset_arch: arm
            goarm: 7
```

Reply:
This also uses arm in `goarch` and will result in gfc error: `-marm` which I pointed at #46 please try from your side you'll understand :(